<?xml version="1.0" ?>
<launch>

	<!-- GENERAL ARGUMENTS -->
	<arg name="robot" default="panda"/>
	<arg name="tool" default="micro2"/>
	<arg name="simulation" default="false"/>

	<!-- ROBOT SETUP -->
	<include file="$(find robot_arm_tools)/launch/Setup_$(arg robot).launch">
		<arg name="load_$(arg tool)" value="true"/>
		<arg name="simulation" value="$(arg simulation)"/>
	</include>

	<!-- MEASUREMENT SERVER -->
	<node pkg="robot_arm_acoustic" type="SoundMeasurementServer.py" name="sound_measurement_server" output="screen"/>

	<!-- MAIN NODE -->
	<node pkg="robot_arm_acoustic" type="robot_arm_acoustic_directivity_node" name="robot_arm_acoustic_directivity_node" output="screen">
		<param name="measurementServerName" value="/sound_measurement_server"/>
		<param name="storageFolderName" value="/tmp/DirectivityMeasurements/"/>
		<rosparam command="load" file="$(find robot_arm_acoustic)/config/AcquisitionParameters.yaml"/>
	</node>

</launch>